composer.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
I didnt fix anything in here, i just deleted the file and regenerated it using `composer install`